pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdFunctions.java

```diff
@@ -911,7 +923,8 @@ private void parseFullyQualifiedFunctionName(String fqfn, FunctionConfig functio
     }
 
     protected static void startLocalRun(org.apache.pulsar.functions.proto.Function.FunctionDetails functionDetails,
-                                        int parallelism, String brokerServiceUrl, String userCodeFile, PulsarAdmin admin)
+            int parallelism, String brokerServiceUrl, String clientAuthPlugin, String clientAuthParam, boolean useTls, boolean allowInsecureConnection,
+            String userCodeFile, PulsarAdmin admin)
             throws Exception {
 
         String serviceUrl = admin.getServiceUrl();
```

Comment:
I think we can get TLS options excluding `useTls` from `admin.getClientConfigData()` as well.